incubator/vaultingkube/templates/deployment.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: apps/v1
```

Comment:
apps/v1beta2

Reply:
Unless I'm mistaken, I understood that this repository aims to support the latest stable version of k8s, and I understood from the reference documentation that the api version of `Deployment` is `v1` https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.9/#deployment-v1-apps
Could you please explain why i'm wrong :)

Reply:
As a side note, I have only seen the idea of supporting the latest stable version of k8s api in an issue and not in any docs (the docs only say `Do not leverage alpha features of Kubernetes` which does not precise which stable versions of k8s should be supported).

Reply:
We official support the current and previous minor versions. CI is not at 1.9 yet either.

See https://github.com/kubernetes/charts/blob/master/REVIEW_GUIDELINES.md#compatibility

Reply:
Done :)